src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  visibleGoods: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ visibleGoods }) => {
```

Comment:
Pass the correct props
```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => {
```